lms/envs/dev_with_worker.py

```diff
@@ -3,29 +3,70 @@
 requirement of a celery worker running in the background to process
 celery tasks.
 
-The worker can be executed using:
+When testing locally, run lms/cms with this settings file as well, to test queueing
+of tasks onto the appropriate workers.
 
-django_admin.py celery worker
+In two separate processes on devstack:
+    paver devstack lms --settings=dev_with_worker
+    ./manage.py lms celery worker --settings=dev_with_worker
 """
 
 # We intentionally define lots of variables that aren't used, and
 # want to import all variables from base settings files
 # pylint: disable=wildcard-import, unused-wildcard-import
 
 from lms.envs.dev import *
```

Comment:
Can we:
1. import from `devstack` instead of `dev`?  I don't know if anyone really maintains dev anymore.
2. once #1 is done, you won't need to duplicate the definitions of the settings below, including `SERVICE_VARIANT`, `CONFIG_PREFIX`, `CELERY_QUEUES`, etc, etc.  Since those are imported into devstack.py from aws.py.